code/__DEFINES/span.dm

```diff
@@ -20,6 +20,7 @@
 #define span_blueteamradio(str) ("<span class='blueteamradio'>" + str + "</span>")
 #define span_bold(str) ("<span class='bold'>" + str + "</span>")
 #define span_boldannounce(str) ("<span class='boldannounce'>" + str + "</span>")
+#define span_bolddanger(str) ("<span class='bolddanger'>" + str + "</span>")
```

Comment:
Is `bolddanger` is even in the style sheet, what does it look like in game (both light and dark mode)?

Reply:
It in fact _is_, but only in light mode. Which is an easy fix.